game/dota_addons/dota_imba/scripts/vscripts/hero/hero_silencer/imba_arcane_curse.lua

```diff
@@ -129,24 +137,21 @@ end
 
 function modifier_imba_arcane_curse_debuff:OnAbilityExecuted( params )
 	if IsServer() then
-		local exception = {
-			["invoker_quas"] = true,
-			["invoker_wex"] = true,
-			["invoker_exort"] = true,
-		}
 		if ( not params.ability:IsItem() ) and ( params.unit == self.parent ) and ( not exception[params.ability:GetName()] ) then
 			-- Only extend duration of Toggle abilities when they are turned on
 			-- OnAbilityExecuted is ran before the toggle completes, so 'true' = we are about to turn it off
+			if CheckExceptions(params.ability) then
+				return
+			end
 			if params.ability:IsToggle() and params.ability:GetToggleState() then
 				return
 			end
-
 			self:SetDuration( self:GetRemainingTime() + self.penalty_duration, true )
 			self:IncrementStackCount()
 		end
 	end
 end
 
 function modifier_imba_arcane_curse_debuff:GetModifierMoveSpeedBonus_Percentage( params )
-	return self.curse_slow
+	return -self.curse_slow
```

Comment:
I changed the KV AbilitySpecial slow thing to be a positive number (since it seems that's the way it's done everywhere else), hence me changing this line.